source/fundamentals/class-mapping.txt

```diff
@@ -172,16 +172,18 @@ stored in the collection as follows:
 Mapping with Constructors
 -------------------------
 
-By default the {+driver-short+} requires a class to have a constructor with no 
-arguments in order to automatically create a class map. However, you can
-instruct the driver to automatically map the class using a constructor that takes one
-or more arguments.
+By default, the {+driver-short+} can automatically map a class only if the class has
+a constructor with no arguments. If you want the driver to use a constructor that accepts
+one or more arguments, you can add the ``[BsonConstructor]`` attribute to the constructor.
+In this case, the driver parses the expression tree to determine how to map the   
+constructor arguments to class properties or fields.
```

Comment:
```suggestion
constructor arguments to class properties.
```

Reply:
It's less common, but technically the arguments can also map to fields instead of properties.